dbt_subprojects/tokens/models/prices/viction/prices_viction_tokens.sql

```diff
@@ -0,0 +1,24 @@
+{% set blockchain = 'viction' %}
+
+{{ config(
+    schema = 'prices_' + blockchain,
+    alias = 'tokens',
+    materialized = 'table',
+    file_format = 'delta',
+    tags = ['static']
+    )
+}}
+
+SELECT
+    token_id
+    , '{{ blockchain }}' as blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('tomo-tomochain', 'WVIC', 0xC054751BdBD24Ae713BA3Dc9Bd9434aBe2abc1ce, 18)
```

Comment:
FYI, `tomochain` is now `viction`:
https://api.coinpaprika.com/v1/coins/tomo-tomochain

Reply:
good callout, relevant blog post:
https://blog.viction.xyz/introducing-viction/